tools/icqsol_rotate_shape/test-data/cylinder_with_field.vtkascii

```diff
@@ -0,0 +1,129 @@
+0.85 -0.153553 -0.0535534 0.5 -0.26194 0.108658 0.85 -0.26194 0.108658 
+0.5 -0.3 0.3 0.85 -0.3 0.3 0.5 -0.26194 0.491342 
+0.85 -0.26194 0.491342 0.5 -0.153553 0.653553 0.85 -0.153553 0.653553 
+0.5 0.00865828 0.76194 0.85 0.00865828 0.76194 0.5 0.2 0.8 
+0.85 0.2 0.8 0.5 0.391342 0.76194 0.85 0.391342 0.76194 
+0.5 0.553553 0.653553 0.85 0.553553 0.653553 0.5 0.66194 0.491342 
+0.85 0.66194 0.491342 0.5 0.7 0.3 0.85 0.7 0.3 
+0.5 0.66194 0.108658 0.85 0.66194 0.108658 0.5 0.553553 -0.0535534 
+0.85 0.553553 -0.0535534 0.5 0.391342 -0.16194 0.85 0.391342 -0.16194 
+
+POLYGONS 64 256
+3 0 1 2 
+3 4 35 34 
+3 34 3 4 
+3 5 37 36 
+3 0 2 6 
+3 7 38 35 
+3 35 4 7 
+3 5 39 37 
+3 0 6 8 
+3 9 40 38 
+3 38 7 9 
+3 5 41 39 
+3 0 8 10 
+3 11 42 40 
+3 40 9 11 
+3 5 43 41 
+3 0 10 12 
+3 13 44 42 
+3 42 11 13 
+3 5 45 43 
+3 0 12 14 
+3 15 46 44 
+3 44 13 15 
+3 5 47 45 
+3 0 14 16 
+3 17 48 46 
+3 46 15 17 
+3 5 49 47 
+3 0 16 18 
+3 19 50 48 
+3 48 17 19 
+3 5 51 49 
+3 0 18 20 
+3 21 52 50 
+3 50 19 21 
+3 5 53 51 
+3 0 20 22 
+3 23 54 52 
+3 52 21 23 
+3 5 55 53 
+3 0 22 24 
+3 25 56 54 
+3 54 23 25 
+3 5 57 55 
+3 0 24 26 
+3 27 58 56 
+3 56 25 27 
+3 5 59 57 
+3 0 26 28 
+3 29 60 58 
+3 58 27 29 
+3 5 61 59 
+3 0 28 30 
+3 31 62 60 
+3 60 29 31 
+3 5 63 61 
+3 0 30 32 
+3 33 64 62 
+3 62 31 33 
+3 5 65 63 
+3 0 32 1 
+3 3 34 64 
+3 64 33 3 
+3 5 36 65 
+
+POINT_DATA 66
+NORMALS Normals float
+-1 0 0 -1 0 0 -1 0 0 
+0 0.0661589 -0.997809 0 -0.320722 -0.947173 1 0 0 
+-1 0 0 0 -0.658776 -0.752339 -1 0 0 
+0 -0.896537 -0.442968 -1 0 0 0 -0.997809 -0.0661589 
+-1 0 0 0 -0.947173 0.320722 -1 0 0 
+0 -0.752339 0.658776 -1 0 0 0 -0.442968 0.896537 
+-1 0 0 0 -0.0661589 0.997809 -1 0 0 
+0 0.320722 0.947173 -1 0 0 0 0.658776 0.752339 
+-1 0 0 0 0.896538 0.442968 -1 0 0 
+0 0.997809 0.0661589 -1 0 0 0 0.947173 -0.320722 
+-1 0 0 0 0.752339 -0.658776 -1 0 0 
+0 0.442968 -0.896537 0 -0.0661588 -0.997809 0 -0.442968 -0.896537 
+1 0 0 1 0 0 0 -0.752339 -0.658776 
+1 0 0 0 -0.947173 -0.320722 1 0 0 
+0 -0.997809 0.0661589 1 0 0 0 -0.896537 0.442968 
+1 0 0 0 -0.658776 0.752339 1 0 0 
+0 -0.320722 0.947173 1 0 0 0 0.0661589 0.997809 
+1 0 0 0 0.442968 0.896537 1 0 0 
+0 0.752339 0.658776 1 0 0 0 0.947173 0.320722 
+1 0 0 0 0.997809 -0.0661589 1 0 0 
+0 0.896538 -0.442968 1 0 0 0 0.658776 -0.752339 
+1 0 0 0 0.320722 -0.947173 1 0 0 
+
+FIELD FieldData 1
+my_field 1 66 double
+1.19292 1.19292 1.20573 1.3179 1.62563 1.3179 1.21993 1.45051 1.15534 
+1.16035 1.13279 1.0751 1.15534 1.16035 1.21993 1.45051 1.20573 1.62563 
+1.19292 1.3179 1.0824 0.902456 1.00774 0.681662 0.967386 0.577466 0.954762 
+0.547018 0.967386 0.577466 1.00774 0.681662 1.0824 0.902456 1.19292 1.20573 
+1.3179 1.62563 1.21993 1.45051 1.15534 1.16035 1.13279 1.0751 1.15534 
+1.16035 1.21993 1.45051 1.20573 1.62563 1.19292 1.3179 1.0824 0.902456 
+1.00774 0.681662 0.967386 0.577466 0.954762 0.547018 0.967386 0.577466 1.00774 
+0.681662 1.0824 0.902456 
```

Comment:
Done.

$ ls -ltr
icqsol_{translate,rotate}_shape/test-data/cylinder_with_field.vtkascii

lrwxr-xr-x  1 pletzer  staff  44 Dec  3 10:56
icqsol_translate_shape/test-data/cylinder_with_field.vtkascii ->
../../test-data/cylinder_with_field.vtkascii

lrwxr-xr-x  1 pletzer  staff  44 Dec  3 10:57
icqsol_rotate_shape/test-data/cylinder_with_field.vtkascii ->
../../test-data/cylinder_with_field.vtkascii

On Thu, Dec 3, 2015 at 9:49 AM, Greg Von Kuster <notifications@github.com>
wrote:

> In tools/icqsol_rotate_shape/test-data/cylinder_with_field.vtkascii
> <https://github.com/gregvonkuster/galaxy-csg/pull/18#discussion_r46577358>
> :
>
> > +1 0 0 0 0.442968 0.896537 1 0 0
> > +0 0.752339 0.658776 1 0 0 0 0.947173 0.320722
> > +1 0 0 0 0.997809 -0.0661589 1 0 0
> > +0 0.896538 -0.442968 1 0 0 0 0.658776 -0.752339
> > +1 0 0 0 0.320722 -0.947173 1 0 0
> > +
> > +FIELD FieldData 1
> > +my_field 1 66 double
> > +1.19292 1.19292 1.20573 1.3179 1.62563 1.3179 1.21993 1.45051 1.15534
> > +1.16035 1.13279 1.0751 1.15534 1.16035 1.21993 1.45051 1.20573 1.62563
> > +1.19292 1.3179 1.0824 0.902456 1.00774 0.681662 0.967386 0.577466 0.954762
> > +0.547018 0.967386 0.577466 1.00774 0.681662 1.0824 0.902456 1.19292 1.20573
> > +1.3179 1.62563 1.21993 1.45051 1.15534 1.16035 1.13279 1.0751 1.15534
> > +1.16035 1.21993 1.45051 1.20573 1.62563 1.19292 1.3179 1.0824 0.902456
> > +1.00774 0.681662 0.967386 0.577466 0.954762 0.547018 0.967386 0.577466 1.00774
> > +0.681662 1.0824 0.902456
>
> @pletzer <https://github.com/pletzer> To save space in the repository,
> there is a test-data directory at the root level that contains files that
> are symlinked as inputs for various tools. Instead of using this additional
> file as a test input, this test-data directory should have a symlink to
> ../../cylinder_colored.vtkascii or some other appropriate input. The output
> file should be kept in this tool's test-data directory as you have it, but
> it should be the result of rotating ../../cylinder_colored.vtkascii.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gregvonkuster/galaxy-csg/pull/18/files#r46577358>.
>



-- 
Alexander Pletzer
+1 609 799 7999 cell
alexander@gokliya.net